distrho/src/DistrhoPluginLV2.cpp

```diff
@@ -169,7 +169,7 @@ class PluginLv2
     void lv2_activate()
     {
 #if DISTRHO_PLUGIN_WANT_TIMEPOS
-        std::memset(&fTimePosition, 0, sizeof(TimePosition));
+        fTimePosition = TimePosition();
```

Comment:
I really dislike this styling..
I would preferably add a `reset` function on the TimePosition struct.

Reply:
How about `clear` as a name instead?
(smart pointers use `reset()`, and can make this a source of silly mistakes)